src/Process.js

```diff
@@ -77,10 +81,12 @@ Process.prototype.stream = T.func([], TDuplexStream, "process.stream").of(functi
 /**
  * Kills a running Process
  *
- * @param {String} signal to kill the process
+ * @param {MaybeExitCodes} maybeExitCodes that are valid
  */
-Process.prototype.kill = T.func([], T.Object, "process.kill").of(function() {
-    return this.instance.kill();
+Process.prototype.kill = T.func([MaybeExitCodes], T.Object, "process.kill").of(function(maybeExitCodes) {
```

Comment:
das problem wasde has liegt an deiner typisierung: Tcomb erwartet, dass du den parameter in dem call explizit setzt, also dass du bei maybe `process.kill(null)` callst. Das is auch der grund, warum bei death keine tcomb typisierung is, heißt, du muss hier einfach das T.func() um die funktion wegmachen und dann sollte dass das gewesen sein. Typchecks sind ja durch ExitCodes.create() trotzdem vorhanden. sollte das problem lösen